plat/kvm/Makefile.uk

```diff
@@ -86,8 +86,12 @@ endif
 ## Architecture library definitions for arm64
 ##
 ifeq ($(findstring y,$(CONFIG_KVM_KERNEL_SERIAL_CONSOLE) $(CONFIG_KVM_DEBUG_SERIAL_CONSOLE)),y)
+ifeq ($(CONFIG_KVM_VMM_FIRECRACKER),y)
```

Comment:
Why does it need to be condittioned by the VMM?

Reply:
Because I forgot to update it, thanks